common/libs/wallet/deterministic/trezor.ts

```diff
@@ -2,24 +2,51 @@ import BN from 'bn.js';
 import EthTx, { TxObj } from 'ethereumjs-tx';
 import { addHexPrefix } from 'ethereumjs-util';
 import { stripHexPrefixAndLower, padLeftEven } from 'libs/values';
-import TrezorConnect from 'vendor/trezor-connect';
-import { DeterministicWallet } from './deterministic';
+import TC from 'vendor/trezor-connect';
+import { HardwareWallet, ChainCodeResponse } from './hardware';
 import { getTransactionFields } from 'libs/transaction';
 import mapValues from 'lodash/mapValues';
-
 import { IFullWallet } from '../IWallet';
 import { translateRaw } from 'translations';
+import EnclaveAPI, { WalletTypes } from 'shared/enclave/client';
 
 export const TREZOR_MINIMUM_FIRMWARE = '1.5.2';
+const TrezorConnect = TC as any;
```

Comment:
I should probably just d.ts this hot boi.

Reply:
Agreed